devops/openshift/workers-deploy.yml

```diff
@@ -77,6 +77,18 @@ objects:
                 requests:
                   cpu: ${CPU_REQUEST}
                   memory: ${MEMORY_REQUEST}
+  - apiVersion: autoscaling/v1
+    kind: HorizontalPodAutoscaler
```

Comment:
Should we consider this for StatefulSet as well?

As the horizontal pod autoscaling is not in the scope of the ticket, we can discuss with team to either add this to ticket or do the autoscaling as a seperate effort considering if we need to include the statefulset or not and any further analysis required including the testing of this capability.

Reply:
Aslo disruptionBudget is missing for this deployment config. Any particular reason?

Reply:
Nice catch I have added it now